fem/pbilinearform.hpp

```diff
@@ -295,15 +340,84 @@ class ParMixedBilinearForm : public MixedBilinearForm
    {
       trial_pfes = trial_fes;
       test_pfes  = test_fes;
+      keep_nbr_block = false;
    }
 
+   /** When set to true and the ParBilinearForm has interior face integrators,
+       the local SparseMatrix will include the rows (in addition to the columns)
+       corresponding to face-neighbor dofs. The default behavior is to disregard
+       those rows. Must be called before the first Assemble call. */
+   void KeepNbrBlock(bool knb = true) { keep_nbr_block = knb; }
```

Comment:
`ParBilinearForm` -> `ParMixedBilinearForm`

Reply:
Fixed in https://github.com/mfem/mfem/pull/4659/commits/110720dd04a743ced5b17174d9cbc8f9044ab3a3 .